service-commands/scripts/.env/bin/migrate

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -e
+
+GIT_URL=git@github.com:joaquincasares/.env.git
```

Comment:
should this be customizable or hardcoded to yours?

Reply:
100% customizable since we should each be tracking our own dotfiles.

Updated so we don't assume you should use mine.